types/architect__sandbox/architect__sandbox-tests.ts

```diff
@@ -26,7 +26,7 @@ sandbox.events.end(() => {});
 sandbox.http.start().then(() => {});
 sandbox.http.start({}).then(() => {});
 sandbox.http.start({ quiet: true }).then(() => {});
-sandbox.http.start({ port: '4444' }).then(() => {});
+sandbox.http.start({ port: 4444 }).then(() => {});
```

Comment:
Never realized GH would only let me comment on changed lines, not on any line on a changed file, but ANYWAY I think tests are failing bc L19? `sandbox.events.start({ port: '4444' }).then(() => {});`